batch/batch/front_end/front_end.py

```diff
@@ -1786,21 +1830,15 @@ async def parse_error(msg):
     if end is not None and start > end:
         return await parse_error('Invalid search; start must be earlier than end.')
 
-    where_conditions = ["billing_projects.`status` != 'deleted'"]
-    where_args = []
+    where_conditions = [
+        "billing_projects.`status` != 'deleted'",
+        "billing_timestamp >= %s",
```

Comment:
I think this should be `billing_date`

Reply:
I messed up by forgetting to switch the name of the column (got the variable name, but not the actual table schema name). We can fix this by either online renaming the column, but that means the two triggers will be out of sync and fail for potentially a second assuming no issues in the migration. Or we do a quick offline migration and guarantee no issues. Thoughts?

Reply:
Ah ok. The rename would be instantaneous? If it's basically just a blip then I think safest to do it offline and nip it in the bud, otherwise I think whatever is simplest and the least headache to keep things consistent.

Reply:
My guess is it would be quick, but it will block the entire table. I think we just take it offline in case there's a problem we don't bork the system.

Reply:
Ok. I think if it would be quick better to get it out of the way sooner rather than later.